### DISCUSSION TITLE
language translate recipes

### DISCUSSION THREAD

**Discussion by gilbrotheraway** [mealie-recipes/mealie#4523] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

please add the option to translate the recipes for non english speaking public with openai or local whisper

### Please provide a concise description of the feature that would resolve your issue.

translation

### Please consider and list out some caveats or tradeoffs made in your design decision

it would greatly expand the public

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by benmccann** [comment]
This looks the same as https://github.com/mealie-recipes/mealie/discussions/3011
